src/foam/java/PropertyInfo.js

```diff
@@ -155,6 +155,23 @@ foam.CLASS({
       factory: function() { return this.property.sheetsOutput; },
       documentation: 'The sheetsOutput specifies if property shoud be written to Google Sheet on import. eg on Transaction import in case there is Status column transaction\'s status will be written there'
     },
+    {
```

Comment:
Can we refine PropertyInfo from the NP side, rather than introduce this here?

Reply:
@adamfox-nanopay see foam/java/refinements.js
Then add your PropertyInfo properties via the refines process on the nanopay side.